divvy/cloudfilebrowser.js

```diff
@@ -466,6 +470,16 @@ var server = (function() {
                 this.authHeader(CloudElements.getUTkn(), CloudElements.getOTkn(), tkn), params, cb, cbArgs);
         },
 
+        search: function(tkn, path, keywoard, cb, cbArgs) {
+            var params = {
+                'path': path,
+                'text': keywoard,
+            }
+
+            _server.call('api-v2/hubs/documents/search', 'Get',
```

Comment:
Is `Get` correct here? Should it be `get` or `GET`, instead?

Reply:
@nmallare `Get` is used all through out the code, should I change those too?